examples/keras_io/tensorflow/vision/mirnet.py

```diff
@@ -0,0 +1,505 @@
+"""
+Title: Low-light image enhancement using MIRNet
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Converted to Keras Core by: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/09/11
+Last modified: 2021/09/15
+Description: Implementing the MIRNet architecture for low-light image enhancement.
+Accelerator: GPU
+"""
+"""
+## Introduction
+
+With the goal of recovering high-quality image content from its degraded version, image
+restoration enjoys numerous applications, such as in
+photography, security, medical imaging, and remote sensing. In this example, we implement the
+**MIRNet** model for low-light image enhancement, a fully-convolutional architecture that
+learns an enriched set of
+features that combines contextual information from multiple scales, while
+simultaneously preserving the high-resolution spatial details.
+
+### References:
+
+- [Learning Enriched Features for Real Image Restoration and Enhancement](https://arxiv.org/abs/2003.06792)
+- [The Retinex Theory of Color Vision](http://www.cnbc.cmu.edu/~tai/cp_papers/E.Land_Retinex_Theory_ScientifcAmerican.pdf)
+- [Two deterministic half-quadratic regularization algorithms for computed imaging](https://ieeexplore.ieee.org/document/413553)
+"""
+
+"""
+## Downloading LOLDataset
+
+The **LoL Dataset** has been created for low-light image enhancement.
+It provides 485 images for training and 15 for testing. Each image pair in the dataset
+consists of a low-light input image and its corresponding well-exposed reference image.
+"""
+
+"""shell
+pip install -q git+https://github.com/keras-team/keras-core
+"""
+
+import os
+import random
+import numpy as np
+from glob import glob
+from PIL import Image, ImageOps
+import matplotlib.pyplot as plt
+
+import keras_core as keras
+from keras_core import layers
+
+import tensorflow as tf
```

Comment:
Since it only works with TF it's a good idea to add `os.environ["KERAS_BACKEND"] = "tensorflow" before the keras import